script/mushroom_mapper.rb

```diff
@@ -90,10 +88,7 @@
 
 # Build table of number of observations per genus.
 observations = {}
-for id in Name.connection.select_values(%(
-  SELECT name_id
-  FROM observations
-)) do
+for id in Observation.pluck(:name_id) do
```

Comment:
Would you mind changing the "for x in y" to "y.each do |x|"?  I think it would make rubocop happier.  (All these for loops were written pre-rubocop, sorry.)

Reply:
Done!